jina/logging/profile.py

```diff
@@ -227,6 +227,7 @@ def __init__(
         self._num_update_called = 0
         self._on_done = message_on_done
         self._stop_event = threading.Event()
+        print()
```

Comment:
can u add a comment on top explaining the reason, to avoid people from removing it?

Reply:
What we actually should do on each update of the log line: Check if the content on the current log line is what we wrote there the last time. Only then update, otherwise create a new line.